var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -70,6 +70,11 @@ class Fftw(AutotoolsPackage):
     depends_on('autoconf', type='build', when='+pfft_patches')
     depends_on('libtool', type='build', when='+pfft_patches')
 
+    @property
+    def libs(self):
+        result = find_libraries(['*fftw3'], root=self.prefix, recurse=True)
```

Comment:
`find_libraries` accepts glob characters, but they really aren't needed here as `libfftw3` should always cut it.